contrib/boot-contracts-stateful-prop-tests/tests/pox-4/pox_StackAggregationCommitIndexedSigCommand_Err.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  logCommand,
+  PoxCommand,
+  Real,
+  Stub,
+  Wallet,
+} from "./pox_CommandModel.ts";
+import { Pox4SignatureTopic, poxAddressToTuple } from "@stacks/stacking";
+import { expect } from "vitest";
+import { Cl } from "@stacks/transactions";
+import { bufferFromHex } from "@stacks/transactions/dist/cl";
+import { currentCycle } from "./pox_Commands.ts";
+
+type CheckFunc = (
+  this: StackAggregationCommitIndexedSigCommand_Err,
+  model: Readonly<Stub>,
+) => boolean;
+
+export class StackAggregationCommitIndexedSigCommand_Err implements PoxCommand {
+  readonly operator: Wallet;
+  readonly authId: number;
+  readonly checkFunc: CheckFunc;
+  readonly errorCode: number;
+
+  /**
+   * Constructs a `StackAggregationCommitIndexedSigCommand_Err` to lock uSTX
```

Comment:
The command is to aggregate partically locked uSTX

Reply:
@friedger I will check again all the comments. Apart from the comments, I think the most important part of this PR is [this](https://github.com/stacks-network/stacks-core/blob/feat/pox-4-stateful-prop-tests-check-err/contrib/boot-contracts-stateful-prop-tests/tests/pox-4/err_Commands.ts) file. This guides how the first round of unexpected PoX-4 interaction is tested. Inside this file, you can see multiple generators for the same command. What makes them different is the `check` method (e.g. [this](https://github.com/stacks-network/stacks-core/blob/feat/pox-4-stateful-prop-tests-check-err/contrib/boot-contracts-stateful-prop-tests/tests/pox-4/err_Commands.ts#L76-L91)), along with the expected error code (e.g. [this](https://github.com/stacks-network/stacks-core/blob/feat/pox-4-stateful-prop-tests-check-err/contrib/boot-contracts-stateful-prop-tests/tests/pox-4/err_Commands.ts#L92)). 

Thank you!